src/tasks/DockerRunTaskProvider.ts

```diff
@@ -31,7 +31,6 @@ export class DockerRunTaskProvider extends DockerTaskProvider {
     // TODO: Skip if container is freshly started, but probably depends on language
     protected async executeTaskInternal(context: DockerRunTaskContext, task: DockerRunTask): Promise<void> {
         const definition = cloneObject(task.definition);
-        definition.dockerRun = definition.dockerRun || {};

```

Comment:
Make sure you're invoking the new base getDockerRunOptions method in any of the child class method overrides to avoid the dockerRun property being undefined.